app/partials/sfox/checkout.jade

```diff
@@ -1,34 +1,44 @@
-.buy-sell.mbvl
-  .flex-row.flex-column-tablet(ng-hide="signupCompleted")
+.filter-bar.flex-center.flex-between(capture-key="{'ArrowLeft': moveTab(-1), 'ArrowRight': moveTab(1)}")
```

Comment:
I love selectedTab, and you're even animating with ng-class="{active: selectedTab === tab}". Very nice!